torch/_dynamo/variables/functions.py

```diff
@@ -369,6 +389,7 @@ def get_globals(self):
 
     def bind_args(self, parent, args, kwargs):
         code = self.get_code()
+        tx = parent.output.root_tx
```

Comment:
made on L392, used on L406, move it down? Or even just slap it right into the only line that uses it?

Reply:
yea i can simplify. did this on purpose to copy the pattern for the other 'bind_args'